scripts/report-bundle-size.js

```diff
@@ -4,18 +4,22 @@
 const path = require('path')
 const childProc = require('child_process')
 
+const bundlesNames = ['main', 'client-optional', 'server-renderer', 'ssr', 'request-processor']
+
```

Comment:
Since the report file name is not `stat.json` anymore, we need to have a list of bundles to report to data

Reply:
Good catch btw to update this file too.